src/javascript/app_2/App/Containers/Layout/header.jsx

```diff
@@ -1,27 +1,29 @@
-import PropTypes                from 'prop-types';
-import React                    from 'react';
-import { withRouter }           from 'react-router';
-import { formatMoney }          from '_common/base/currency_base';
+import PropTypes       from 'prop-types';
+import React           from 'react';
+import { withRouter }  from 'react-router';
+import { formatMoney } from '_common/base/currency_base';
 import {
     AccountInfo,
+    DepositButton,
     LoginButton,
+    UpgradeButton,
```

Comment:
@zakibinary Should it be in alphabetic order?

Reply:
Yes, @morteza-binary , thanks for pointing this out.